app/experimenter/nimbus-ui/src/components/HeaderEditExperiment/index.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { getExperiment } from "../../types/getExperiment";
+
+const HeaderEditExperiment = ({
+  experiment,
```

Comment:
I'm pulling in all of the experiment data here rather than just name/slug/status because when we transform this into a form later (post-parity), I believe we'll need to submit with all experiment data.

Reply:
That doesn't appear to be necessary. I just tried in the [playground](https://localhost/api/v5/graphql#query=%23%20query%20%7B%0A%23%20%20%20experiments%20%7B%0A%23%20%20%20%20%20slug%0A%23%20%20%20%20%20id%0A%23%20%20%20%20%20name%0A%23%20%20%20%7D%0A%23%20%7D%0A%0Amutation%20%7B%0A%20%20updateExperiment(input%3A%20%7B%0A%20%20%20%20id%3A%20%221%22%0A%20%20%20%20name%3A%20%22Hey%20hey%20hey%22%0A%20%20%7D)%20%7B%0A%20%20%20%20status%0A%20%20%20%20message%0A%20%20%7D%0A%7D), and the only required input data for `updateExperiment` appears to be the ID. Everything else looks optional:

```tsx
mutation {
  updateExperiment(input: {
    id: "1"
    name: "Hey hey hey"
  }) {
    status
    message
  }
}
```

Reply:
I shall change it based on that conversation earlier too on not turning this into a form :D